src/users/dto/update-user.dto.ts

```diff
@@ -0,0 +1,5 @@
+
+export class UpdateUserDto {
+    name?: string;
```

Comment:
please add validation from "class-validator" libraty
For example: 
@IsString()
name: string